docs/changelog/0.12.0.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: 0.12.0
+---
+
+## New Features 💡
+
+### JSON Syntax
+
+- [#1811](https://github.com/paradedb/paradedb/pull/1811) introduced the ability to pass JSON objects directly to the search query, similar to Elasticsearch's Query DSL. The documentation has been updated to reflect this new query syntax option.
+
+### Index Configuration
```

Comment:
Maybe here we mention #1983 where we no longer append a `_bm25_index` suffix to new index names and instead accept the user's name directly.

Why this is important is that I think it was common for us to suggest to users that they use the same value for `index_name =>` and `table_name =>` and that's no longer the case -- can't have two relations with the same name.